src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -163,6 +161,9 @@ void fillInnerParts() {
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
 
+        studentDetailList.setVisible(false);
+        lessonDetailList.setVisible(false);
```

Comment:
is this setting of visibility to false required? because the list visibility logic should be handled when the contentsplitpane removes or adds the lists (in line 167, 238-243)